test.md

```diff
@@ -107,7 +107,7 @@ culpa qui officia deserunt mollit anim id est laborum.
   - You can specify `org=foo` to use a Github organization other than Docker
 
 - A link to an auto-generated reference page that we pull in during docs builds:
-[/engine/reference/builder/#env](/engine/reference/builder/#env).
+[/dockerfile/reference/#env](/dockerfile/reference/#env).
```

Comment:
Perhaps we should start to consider using `/reference/xxxx` again for URLs instead;

- `/reference/dockerfile/`
- `/reference/commandline/xxx` etc

Reply:
Hum indeed good point

Reply:
Keeping in `/engine/reference/builder/` for now. Let's do the slug change in a follow-up.